app/forms/regulatorsAndDocuments/CharityOtherRegulatorDetailsFormProvider.scala

```diff
@@ -24,7 +24,7 @@ import play.api.data.Forms._
 
 class CharityOtherRegulatorDetailsFormProvider @Inject() extends Mappings {
 
-  private[regulatorsAndDocuments] val validateFields = """^[^@&:)(]+$"""
+  private[regulatorsAndDocuments] val validateFields = "^[a-zA-Z0-9-, ']+$"
```

Comment:
could you please move it to common place as its going repeate, may be part of Mappings and